docs/details/commands/pmm-admin.md

```diff
@@ -508,6 +510,58 @@ PMM communicates with the PMM Server via a PMM agent process.
     `--disable-collectors`
     : Comma-separated list of collector names to exclude from exporter
 
+#### HAProxy
+
+`pmm-admin add haproxy [FLAGS] [NAME]`
+:   Add HAProxy to monitoring.
+
+    FLAGS:
+
+    `--server-url=SERVER-URL`
+    : PMM Server URL in `https://username:password@pmm-server-host/` format
+
+    `--server-insecure-tls`
+    : Skip PMM Server TLS certificate validation.
+
+    `--username=USERNAME`
+    : HAProxy username.
+
+    `--password=PASSWORD`
+    : HAProxy password.
+
+    `--scheme=SCHEME`
+    : Scheme to generate URI to exporter metrics endpoints (http or https).
+
+    `--metrics-path=METRICS-PATH`
+    : Path under which metrics are exposed, used to generate URI (default: /metrics).
+
+    `--listen-port=LISTEN-PORT`
+    : Listen port of external exporter for scraping metrics (Required).
```

Comment:
```suggestion
    : Listen port of haproxy exposing the metrics for scraping metrics (Required).
```

Reply:
@rnovikovP Yes, here https://github.com/percona/pmm-admin/blob/PMM-6797-cli-for-haproxy/commands/management/add_haproxy.go#L130:L130, going to change it to same message like here. Only another occurrence what I know is in pmm inventory in grafana. Where you can see external exporter which was added with haproxy service. Is it OK?